src/emulator/storage/constants.ts

```diff
@@ -0,0 +1,3 @@
+import { randomBytes } from "node:crypto";
+
+export const privateKey = randomBytes(32).toString("hex");
```

Comment:
nit: Call this something more descriptive: `SIGNED_URL_PRIVATE_KEY`. 

I think it might make sense for us to actually hardcode a key (call `randomBytes(32).toString("hex")` in your terminal and copy/paste the output). If we generate a new one each time, signed urls will become invalid if a dev restarts the emulator (probably not uncommon).